app/Http/Middleware/RouteAutoAuthenticate.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\Auth;
+
+class RouteAutoAuthenticate
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @param  string|null  $guard
+     * @return mixed
+     */
+    public function handle($request, Closure $next, $guard = null)
+    {
+        // check request for a user param; use #1 if there isn't one.
+        $user_id = ($request->query('user')) ? $request->query('user') : 1;
```

Comment:
Who is user one? Do we want to hard code that in?

Reply:
User one is setup by the supporting seeds. System will throw an exception if user one doesn't exist.
At this point i just want a default user with the option to fiddle the url to find another one.